src/index.js

```diff
@@ -0,0 +1,54 @@
+import './style.css';
```

Comment:
![image](https://github.com/eamador1/to-do-list/assets/93039359/a68ad30e-266d-4b48-a26f-d2431ad70459)

- [ ] It would be nice if you fix the error which i am facing from your app 😨 Kindly fix it to make your app work as expected 👍

Reply:
That's weird. I will wait for tomorrow to have my coding partner check it out


<img width="1263" alt="Screenshot 2023-08-16 at 22 08 24" src="https://github.com/eamador1/to-do-list/assets/124623610/8b702508-9bef-4be3-a684-58de7234967e">